x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -77,6 +78,7 @@ export function transformUpdateResponseToExternalModel(
       ...(transformedConnector && { connector: transformedConnector }),
       // if externalService is null that means we intentionally updated it to null within ES so return that as a valid value
       ...(externalService !== undefined && { external_service: externalService }),
+      ...(customFields !== undefined && (customFields as unknown as CaseCustomFields)),
```

Comment:
it is mostly because of typescript. If I don't include it, it complains that the field type in `CasesPersistedAttributes` conflicts with `CasesTransformedAttributes`.

Reply:
I tested and I did not get any errors.

Reply:
In this case, did you also remove `customFields` from line 48? 😄 because then they aren't included.

The error only showed for me when I noticed that.

Reply:
Yup, L127 was working for me too 🙌